openslides/core/models.py

```diff
@@ -152,6 +163,25 @@ def need_full_update_for(self, collection_element):
         return True
 
 
+class ProjectionDefault(RESTModelMixin, models.Model):
```

Comment:
Now there is one advantage: If I want to have the default projector id of motions, i could search the ProjectionDefault with name motions and get the id. If the ProjectionDefaults are suborderd under Projector. I have to iterate through all projectors to find the Projector with the given ProjectionDefault. Is this the solution?